### DISCUSSION TITLE
Maintenance auto start/end is not working

### DISCUSSION THREAD

**Discussion by palicepta** [Status-Page/Status-Page#591] [General]
subj.

**Comment by palicepta** [comment]
Also, can you please change the incident timeline structure by last update?

**Comment by herrtxbias** [comment]
Hey @palicepta,
Could you please check your Django Admin Dashboard for any failed jobs within "Background Tasks"? If so, please provide the error message displayed there.

Please also check if your rqworker and rqscheduler are up and running.

**Comment by palicepta** [comment]
Hello [HerrTxbias](https://github.com/HerrTxbias)

I checked the Django dashboard. there are no observed any failed jobs within "Background Tasks"

Do you have any additional ideas?

Thank you in advance!
